okhttp/src/main/java/okhttp3/internal/http/RetryAndFollowUpInterceptor.java

```diff
@@ -153,8 +151,6 @@ public RetryAndFollowUpInterceptor(OkHttpClient client) {
    */
   private boolean recover(IOException e, Transmitter transmitter,
       boolean requestSendStarted, Request userRequest) {
-    transmitter.streamFailed(e);
```

Comment:
This is the most important part of the PR. We’re now tracking failures where they’re thrown, not far away in interceptors.